manifests/r/Rils/TouchPortal/2.3.010/Rils.TouchPortal.locale.en-US.yaml

```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Rils.TouchPortal
+PackageVersion: 2.3.010
+PackageLocale: en-US
+Publisher: Rils

```

Comment:
```suggestion
Publisher: Touch Portal VoF
```
![image](https://user-images.githubusercontent.com/11600822/120531308-65024f80-c3d6-11eb-9803-6ae7c45b6a0d.png)

Reply:
OK, I was confused here because companion apps for desktop app have Rils as publisher